src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -0,0 +1,845 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from safeds.data.tabular.containers import Column, Row, Table, TaggedTable
+from safeds.exceptions import (
+    ColumnIsTargetError,
+    ColumnIsTimeError,
+    IllegalSchemaModificationError,
+    UnknownColumnNameError,
+)
+
+if TYPE_CHECKING:
+    from collections.abc import Callable, Mapping, Sequence
+    from typing import Any
+
+
+class TimeSeries(TaggedTable):
+
+    # ------------------------------------------------------------------------------------------------------------------
+    # Creation
+    # ------------------------------------------------------------------------------------------------------------------
+    @staticmethod
+    def _from_tagged_table(
+        tagged_table: TaggedTable,
+        time_name: str,
+    ) -> TimeSeries:
+        """Create a time series from a tagged table.
+
+        Parameters
+        ----------
+        table : TaggedTable
+            The tagged table.
+        time_name: str
+            Name of the time column.
+
+        Returns
+        -------
+        time_series : TimeSeries
+            the created time series
+
+        Raises
+        ------
+        UnknownColumnNameError
+            If time_name matches none of the column names.
+        Value Error
+            If no feature columns are specified
+        Value Error
+            If time name is in features
```

Comment:
```suggestion
            If time name is the same as the target column name
```
Do you mean the error in line 67?